mlflow/entities/model_registry/__init__.py

```diff
@@ -1,8 +1,12 @@
 from mlflow.entities.model_registry.registered_model import RegisteredModel
 from mlflow.entities.model_registry.model_version import ModelVersion
+from mlflow.entities.model_registry.registered_model_tag import RegisteredModelTag
+from mlflow.entities.model_registry.model_version_tag import ModelVersionTag
 
 
 __all__ = [
     "RegisteredModel",
     "ModelVersion",
+    "RegisteredModelTag",
+    "ModelVersionTag"
```

Comment:
Hyper nit: Add trailing comma

```suggestion
    "ModelVersionTag",
```